easybuild/easyconfigs/m/MUMPS/MUMPS-5.3.5-foss-2020b-parmetis.eb

```diff
@@ -0,0 +1,27 @@
+name = 'MUMPS'
+version = '5.3.5'
+versionsuffix = '-parmetis'
+
+homepage = 'https://graal.ens-lyon.fr/MUMPS/'
+description = "A parallel sparse direct solver"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['http://mumps.enseeiht.fr/']
+sources = ['%(name)s_%(version)s.tar.gz']
```

Comment:
Aren't the patches from [here](https://github.com/easybuilders/easybuild-easyconfigs/blob/7c4cd2735f43c34c93355f01f9d5a2f93ac218ef/easybuild/easyconfigs/m/MUMPS/MUMPS-5.3.5-intel-2020b-metis.eb#L14) also relevant for this build against ParMETIS?